presto-docs/src/main/sphinx/installation/installation-ambari-original.rst

```diff
@@ -0,0 +1,126 @@
+************************************
+Automated Installation for a Cluster
```

Comment:
Please ignore this file. It's a copy of the original. Changes are in installation-ambari-original.rst